app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,7 @@
 <resources>
 
     <string name="app_name">Meals Made Easy</string>
+    <string name="app_name_no_spaces">MealsMadeEasy</string>
```

Comment:
We don't really need this. I don't remember if we settled on the app name having spaces or not, but you can just use `app_name` instead to make sure we're consistent everywhere.